src/coreclr/pal/src/libunwind/src/arm/Gtrace.c

```diff
@@ -526,7 +526,7 @@ tdep_trace (unw_cursor_t *cursor, void **buffer, int *size)
       break;
 
     case UNW_ARM_FRAME_SYSCALL:
-      printf("XXX1\n");
+      //printf("XXX1\n");
```

Comment:
not familiar with this code, what was the problem?

Reply:
```
[   71s]   In file included from /home/abuild/rpmbuild/BUILD/coreclr-6.0.0/src/coreclr/pal/src/libunwind/src/arm/Ltrace.c:4:
[   71s]   /home/abuild/rpmbuild/BUILD/coreclr-6.0.0/src/coreclr/pal/src/libunwind/src/arm/Gtrace.c: In function '_ULarm_tdep_trace':
[   71s]   /home/abuild/rpmbuild/BUILD/coreclr-6.0.0/src/coreclr/pal/src/libunwind/src/arm/Gtrace.c:529:7: error: incompatible implicit declaration of built-in function 'printf' [-Werror]
[   71s]     529 |       printf("XXX1\n");
[   71s]         |       ^~~~~~
[   71s]   /home/abuild/rpmbuild/BUILD/coreclr-6.0.0/src/coreclr/pal/src/libunwind/src/arm/Gtrace.c:31:1: note: include '<stdio.h>' or provide a declaration of 'printf'
[   71s]      30 | #include <limits.h>
[   71s]     +++ |+#include <stdio.h>
[   71s]      31 |
```

Reply:
Would adding `#include <stdio.h>` at the beginning of the file fix the issue or it would introduce some other errors? 
cc @janvorli

Reply:
I've removed this change from here